.gitignore

```diff
@@ -23,6 +23,7 @@ var/
 .installed.cfg
 *.egg
 non-git/
+*.tar.gz
```

Comment:
Should `*.rpm` be ignored is well?  I think `rpmbuild` would generate a `.src.rpm` file.